packages/venia-ui/lib/components/Accordion/__mocks__/index.js

```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+
+const MockAccordion = jest.fn(({ children, ...props }) => (
```

Comment:
I did this after discussions from peer review; that our components would benefit from more manual mocks. Would like to discuss this further, but provided manual mocks like these is pretty simple, but not necessary.

Reply:
I may have stumbled onto why we haven't embraced this more. It seems some very old Jest bug causes these duplicate mock warnings, because `jest-haste-map` stores these mocks by name and not by path. Given we're a project where we use index files to export public API, I can see how this warning may have scared us away from manual mocks, since all of these module names would be index.

I'm digging for a solution, but nothing seems to be working. As you can see, the tests function fine, there's just this very in your face warning you get the first time you run the suite. We could mock underlying components instead of index exports, but expect we'd run into a similar issue eventually.

Original Bug: https://github.com/facebook/jest/issues/2070
The warning:

```
jest-haste-map: duplicate manual mock found: index
  The following files share their name; please delete one of them:
    * <rootDir>/packages/peregrine/lib/RestApi/__mocks__/index.js
    * <rootDir>/packages/venia-ui/lib/components/Accordion/__mocks__/index.js
```

Reply:
Went ahead and just pulled this out, can disregard.